Tools/symbolicate-ppc.py

```diff
@@ -0,0 +1,95 @@
+#!/bin/python3
```

Comment:
Updated the path and switch to `python` (I think the script works with both python 2 and 3).